src/mina-signer/MinaSigner.ts

```diff
@@ -124,7 +124,7 @@ class Client {
    */
   signFields(fields: bigint[], privateKey: Json.PrivateKey): Signed<bigint[]> {
     let privateKey_ = PrivateKey.fromBase58(privateKey);
-    let signature = sign({ fields }, privateKey_, 'testnet');
+    let signature = sign({ fields }, privateKey_, 'mainnet');
```

Comment:
revert! again, `signFields()` is unrelated to any network and back when this was implemented and internally audited, we agreed on sticking with the `'testnet'` prefix.

just to reiterate: this doesn't mean this can't be used on mainnet. signFields just isn't tied to the network in any way

Reply:
Yeah thank you I got it from the previous review, I just thought since we're aiming to the mainnet and there is not differences then probably better to use 'mainnet'. I didn't know there was an agreement.